halloffame.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=halloffame', 'root', 'password');
```

Comment:
Best to put the DB connection in a function so you can reuse on other pages of your site. Although, side note, you can't unit test DB connections so even after creating a function from it - don't worry about making a unit test for it!